addons/es/verifactu/bill.go

```diff
@@ -122,6 +125,10 @@ func validateInvoice(inv *bill.Invoice) error {
 			),
 			validation.Skip,
 		),
+		validation.Field(&inv.Supplier,
+			validation.By(validateInvoiceSupplier),
+			validation.Skip,
+		),
```

Comment:
Rather than validating the Invoice specifically, I'd try to validate the `org.Party` model in general to check for the forbidden characters.